pkg/build/apk.go

```diff
@@ -174,3 +176,52 @@ func (bc *Context) FixateApkWorld() error {
 
 	return bc.Execute("apk", args...)
 }
+
+func (bc *Context) fixScriptsTar() error {
```

Comment:
`fix` is a generic/broad term, a better name should be used (suggestions are welcomed :sweat_smile: )

Reply:
`normalizeApkScriptsTar()` I think.  Once the APK distributions adopt apk3, this file will go away, so we should make sure the file exists before normalizing it too.

Reply:
Oh, good point! Fixed.